chart/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# apiVersion v1 is Helm 2
+apiVersion: v1
```

Comment:
So, should we support helm2, or just helm3 now?

This chart right now doesn't work out of the box with Helm3:

```
airflow ❯ helm dependency update chart
Error: no repository definition for @stable. Please add them via 'helm repo add'
```

(Looking for if there is a solution to support both)

Reply:
This only seems to be a problem when trying to point helm3 at a file/folder, but when installed from an actual repo, it works fine. Curious...

Reply:
Ah I hadn't done this https://v3.helm.sh/docs/intro/quickstart/#initialize-a-helm-chart-repository